### ISSUE TITLE
Reason for duplicating the kinematics_solver key in both groups?

### ISSUE THREAD

**Issue by gavanderhoorn** [marip8/sampling_kinematics_plugin#4]
Tbf I haven't read the code yet, but I was wondering why the configuration needs to duplicate the solver entry in the `ext_` map: could the value of `robot_group` not be used to retrieve the value of `kinematics_solver`?

```yaml
ext_axis_manipulator:
  [..]
  robot_group: manipulator
  robot_kinematics_solver: ur_kinematics/UR5KinematicsPlugin
manipulator:
  kinematics_solver: ur_kinematics/UR5KinematicsPlugin
  [..]
```

Avoiding duplication would be good: one less place to make mistakes.

**Comment by gavanderhoorn** [comment]
Alternatively a YAML anchor + alias could be used, but that would still need manual changes to the file.

**Comment by marip8** [comment]
I did it this way out of expediency, but I agree it's not the best method. My initial thought is to use the `lookupParam` method to get the `<robot_group>/kinematics_solver` parameter. Is there a better way to access this using some other MoveIt class like `RobotModel` or `JointModelGroup` though?

**Comment by marip8** [comment]
Using the `lookupParam` method seems to work well enough, so I think I'm going to go that route

**Comment by marip8** [comment]
Closing, as this was addressed by #6. Thanks for the suggestion @gavanderhoorn
